src/Parser.php

```diff
@@ -33,8 +33,9 @@ public function fromArray($array)
         return Reader\ArrayReader::read($this, $array);
     }
 
-    public function fromFile($file)
+    public function fromFile($file, $fixEncoding = true)
```

Comment:
Super minor but could we add the type hint here please?
```suggestion
    public function fromFile($file, bool $fixEncoding = true)
```